src/Store/Reducers/CVEsStore.test.js

```diff
@@ -167,9 +167,9 @@ describe('CVE reducer:', () => {
             openedCves: ['cve-1', 'cve-2', 'cve-3'],
             expandCveDescription: false
         };
-        action.type = ActionTypes.CLEAR_VULNERABILITY_STORE;
+        action.type = ActionTypes.CLEAR_CVES_STORE;
```

Comment:
here also `CLEAR_CVES_STORE` to `CLEAR_CVEs_STORE`. or simply change this action type :))

Reply:
the same here, the variable is `CLEAR_CVES_STORE` https://github.com/RedHatInsights/vulnerability-ui/blob/404cc2fbf48019a65b469f4cd13df37c53fd7183/src/Store/ActionTypes.js#L28

Reply:
Oh, it is my fault sorry